src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -66,10 +67,10 @@
             .limitMotorOutputs(1.0, -1.0)
             .noMotorStartupRamping()
             .useSmartMotionControl()
-            .withPositionGains(RobotConstants.PID.AUTO.F,
-                    RobotConstants.PID.AUTO.P,
-                    RobotConstants.PID.AUTO.I,
-                    RobotConstants.PID.AUTO.D)
+            .withPositionGains(RobotConstants.PID.AUTO_STRAIGHT.F,
```

Comment:
im not sure we need or should have different PID constants for stright vs 'not straight'. But we'll know for sure when we do tuning